bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/eclipse/common/core/model/MistakeType.java

```diff
@@ -13,9 +18,12 @@
 import edu.kit.kastel.eclipse.common.core.model.rule.PenaltyRule;
 
 public class MistakeType implements IMistakeType {
+	private static final Locale DEFAULT_LOCALE = Locale.GERMAN;
```

Comment:
Add this to the Class ArtemisConstants (in the API Module (as far as I remember))